### PR TITLE
Feature/port to jdk11

### PR DESCRIPTION
**Pull request by jaserud**
Port to JDK 11.

Reference issue #158

### CODE REVIEW

**Comment by jaserud** [comment]
Actually I missed that the Docker versions for ava needs to updaed.

**Comment by rtisma** [comment]
Tested the client using this PRs code and against score-client-1.6.4 (Java 1.8) , and all functionality matches

The illegal reflective access warnings need to be removed
```
Starting...WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.springframework.cglib.core.ReflectUtils$1 (jar:file:/home/rtisma/Documents/prs/score-161/score-client/target/score-client-1.6.5-SNAPSHOT/lib/score-client.jar!/BOOT-INF/lib/spring-core-4.3.3.RELEASE.jar!/) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of org.springframework.cglib.core.ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

File: score-client/src/main/bin/score-client [hunk 1]
```diff
@@ -35,10 +35,16 @@ fi
 
 if [[ "$_java" ]]; then
     version=$("$_java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
-    if [[ "$version" > "1.8" ]]; then
-        :
+
+    regex="^1?\.." #matches 1.xx java versioning
```
Comment (rtisma) [inline score-client/src/main/bin/score-client]: change the regex to
`^1[1-9][\.]{0,1}`

File: score-client/src/main/bin/score-client [hunk 2]
```diff
@@ -35,10 +35,16 @@ fi
...
+    if [[ $version =~ $regex ]]; then 
+        versionMajor=$(echo $version | awk -F '.' '{print $2}') 
     else
-        echo Java 1.8 required for SCORe Client. Current version is $version
+	versionMajor=$(echo $version | awk -F '.' '{print $1}') 
+    fi
+    
+    if [[ $versionMajor -lt 11 ]]; then
+        echo Java 11 required for SCORe Client. Current version is $version
```
Comment (rtisma) [inline score-client/src/main/bin/score-client]: you can merge of all this to say, if the regex is not matches, then error out and indicate the version that caused the error

**Comment by rtisma** [comment]
also, Test are failing. Seems like the score-fs tests.

**Comment by rtisma** [comment]
only the build job for score needs to be updated to use java 11, the rest still need to use java 1.8

**Pull request merged**